doc/api/tls.markdown

```diff
@@ -493,6 +494,24 @@ Example:
 If the peer does not provide a certificate, it returns `null` or an empty
 object.
 
+### tlsSocket.getProtocol()
+
+Returns a string containing the negotiated SSL/TLS protocol version of the
+current connection. `'unknown'` will be returned for connected sockets that have
```

Comment:
Oh, okay then. Since we claim that it would return unknown, I thought that it would have been better to have a test to confirm that.